services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -54,7 +54,25 @@ export type ActionDetails = {
     lockMessage?: string;
     version?: number;
 };
-
+let totalActions = 0;
+export const clearActionsNumber = (): void => {
+    totalActions = 0;
+    const elem = document.getElementsByClassName('mdc-drawer-sidebar-header-title');
+    if (elem[0]) {
+        elem[0].innerHTML = 'Planned Actions';
+    }
+};
+export const updateActionsNumber = (added: boolean): void => {
+    added ? (totalActions += 1) : (totalActions -= 1);
+    const elem = document.getElementsByClassName('mdc-drawer-sidebar-header-title');
```

Comment:
Also, just a tip. It's not recommended to use this way to update some components or text in the app. 

they are already somewhere in the code, and you should do the updating there.

Reply:
I changed it to be directly edited where it is defined.